css/css-cascade/scope-nesting.html

```diff
@@ -97,25 +107,24 @@
 <template id=test_inner_nest>
   <div>
     <style>
-      @scope (.a) {
-        & + & {
-          z-index:1;
+      @scope (#div) {
+        & {
+          z-index: 1;
+          & {
+            z-index: 2;
+          }
         }
       }
     </style>
-    <div class=a>
-      <div id=inner1 class=a></div>
-      <div id=inner2 class=a></div>
-    </div>
+    <div id=div></div>
   </div>
 </template>
 <script>
 test((t) => {
   t.add_cleanup(() => main.replaceChildren());
   main.append(test_inner_nest.content.cloneNode(true));
 
-  assert_equals(getComputedStyle(inner1).zIndex, 'auto');
-  assert_equals(getComputedStyle(inner2).zIndex, '1');
+  assert_equals(getComputedStyle(div).zIndex, '2');
 }, 'Nesting-selector in the scope\'s <stylesheet>');
```

Comment:
Is the new goal of the test to verify that scoped element are *inclusive* descendant of the scoping root ? If so, should we rename the test ?